packetbeat.dev.yml

```diff
@@ -1,43 +1,41 @@
-################### Packetbeat Configuration Example ######################
+################### Packetbeat Configuration Example ##########################
 
 # This file contains an overview of various configuration settings. Please consult
 # the docs at https://www.elastic.co/guide/en/beats/packetbeat/current/_configuration.html
 # for more details.
 
-# The Packetbeat agent works by sniffing the network traffic between your
+# The Packetbeat shipper works by sniffing the network traffic between your
```

Comment:
I think we should keep the word "agent" here as otherwise it will be confused with the Shipper below.